miasm/analysis/dse.py

```diff
@@ -258,11 +258,12 @@ def add_lib_handler(self, libimp, namespace):
 
         # lambda cannot contain statement
         def default_func(dse):
-            fname = b"%s_symb" % force_bytes(libimp.fad2cname[dse.jitter.pc])
+            fname = libimp.fad2cname[dse.jitter.pc]
+            fname = b"%s_symb" % force_bytes(str(fname))
```

Comment:
When `fname` is already a `bytes` object, this would raise a `TypeError`, which is probably not what you want.